docs/content/en/docs/tasks/packages/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Package management"
+date: 2022-04-12
+weight: 20
+description: >
+  Common tasks for managing curated packages.
+---
+
+The main goal of EKS Anywhere curated packages is to make it easy to install, configure and maintain operational components in an EKS Anywhere cluster. EKS Anywhere curated packages offers to run secure and tested operational components on EKS Anywhere clusters. Please check out [package controller]({{< relref "../../concepts" >}}) for more details.
```

Comment:
This needs to be updated to link to the page under https://github.com/aws/eks-anywhere/pull/1877.